exporter/model.go

```diff
@@ -288,19 +306,36 @@ func (r *resource) ImportResource(ic *importContext) {
 		if apiVersion != "" {
 			ctx = context.WithValue(ctx, common.Api, apiVersion)
 		}
-		if dia := pr.ReadContext(ctx, r.Data, ic.Client); dia != nil {
-			log.Printf("[ERROR] Error reading %s#%s: %v", r.Resource, r.ID, dia)
-			return
+		// TODO: rewrite to retries package...
```

Comment:
Is this something we plan to do in future? Just curious if retries package would require refactoring to support this.

Reply:
Miles recommends it, but I need to play with it more